client/settings/reporting-settings/constants.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export const exportLanguageOptions = [
```

Comment:
I wonder if it's really a constant since it has a few moving parts such as `wcpaySettings.locale.code`. Would it be better to export this array from a function, eg. `getExportLanguageOptions`?